src/main/scala/com/microsoft/sqlserver/jdbc/spark/connectors/ReliableSingleInstanceStrategy.scala

```diff
@@ -157,28 +157,32 @@ object ReliableSingleInstanceStrategy extends  DataIOStrategy with Logging {
 
   /**
    * utility function to get all global temp table names as a list.
-   * @param appId appId used as prefix of tablename
-   * @param nrOfPartitions number of paritions in dataframe used as suffix
+   * @param appId appId used as prefix of staging table name
+   * @param dbtable destination table name used as prefix of temp staging table name
+   * @param nrOfPartitions number of partitions in dataframe used as suffix
    */
   private def getStagingTableNames(
                 appId: String,
+                dbtable: String,
                 nrOfPartitions: Int): IndexedSeq[String] = {
     val stagingTableList = for (index <- 0 until nrOfPartitions) yield {
-      getStagingTableName(appId, index)
+      getStagingTableName(appId, dbtable, index)
     }
     stagingTableList
   }
 
   /**
    * utility function to create a staging table name
-   * @param appId appId used as prefix of tablename
+   * @param appId appId used as prefix of table name
+   * @param dbtable destination table name used as prefix of temp staging table name
```

Comment:
is the Fully Qualified table name or just table name. Important to clarify.

Reply:
fully qualified table name will be used. Fixed.